spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,204 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `atan2()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> unitless/input.scss
+a {b: atan2(1, -10)}
+
+<===> unitless/output.css
+a {
+  b: 174.2894068625deg;
+}
+
+<===>
+================================================================================
+<===> compatible_units/input.scss
+a {b: atan2(1cm, -10mm)}
+
+<===> compatible_units/output.css
+a {
+  b: 135deg;
+}
+
+<===>
+================================================================================
+<===> operation/math/input.scss
+a {b: atan2(10 + 6, 8 - 1)}
+
+<===> operation/math/output.css
+a {
+  b: 66.3706222693deg;
+}
+
+<===>
+================================================================================
+<===> operation/nested_function/input.scss
+a {b: atan2(max(2mm, 9mm), min(1cm, 3cm))}
+
+<===> operation/nested_function/output.css
+a {
+  b: 41.9872124958deg;
+}
+
+<===>
+================================================================================
+<===> operation/simplification/input.scss
+a {
+  b: atan2(3px - 1px + var(--c), 4px + 10px);
+}
+
+<===> operation/simplification/output.css
+a {
+  b: atan2(2px + var(--c), 14px);
+}
+
+<===>
+================================================================================
+<===> preserved/y_arg/percentage/input.scss
+a {b: atan2(1.5px + 5%, 4px)}
+
+<===> preserved/y_arg/percentage/output.css
+a {
+  b: atan2(1.5px + 5%, 4px);
+}
+
+<===>
+================================================================================
+<===> preserved/x_arg/percentage/input.scss
+a {b: atan2(4px, 1.5px + 5%)}
+
+<===> preserved/x_arg/percentage/output.css
+a {
+  b: atan2(4px, 1.5px + 5%);
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  b: atan2(var(--c));
+}
+
+<===> preserved/variable/output.css
+a {
+  b: atan2(var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/unknown_units/input.scss
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===> preserved/unknown_units/output.css
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===>
+================================================================================
+<===> error/incompatible_units/input.scss
+a {b: atan2(1, 1px)}
+
+<===> error/incompatible_units/error
+Error: 1 and 1px are incompatible.
+  ,
+1 | a {b: atan2(1, 1px)}
+  |             ^ 1
+  |                === 1px
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/one_arg/input.scss
```

Comment:
We consistently call this `too_few_args` elsewhere. It's important to pay attention to the surrounding style and try to match it whenever possible. It both saves you time that you would otherwise have spent coming up with new conventions, and helps keep the codebase (or in this case test suite) consistent and easy to navigate.